configuration/appframework/referenceapplication/visitActions_extension.json

```diff
@@ -52,13 +52,32 @@
         "extensionPointId": "patientDashboard.visitActions",
         "type": "link",
         "label": "referenceapplication.app.capturevitals.title",
-        "url": "htmlformentryui/htmlform/enterHtmlFormWithSimpleUi.page?patientId={{patient.uuid}}&visitId={{visit.uuid}}&definitionUiResource=referenceapplication:htmlforms/vitals.xml",
+        "url": "htmlformentryui/htmlform/enterHtmlFormWithSimpleUi.page?patientId={{patient.uuid}}&visitId={{visit.uuid}}&definitionUiResource=file%3Aconfiguration%2Fhtmlforms%2Fvitals.xml",
         "icon": "icon-vitals",
         "order": 52,
         "requiredPrivilege": "Task: referenceapplication.vitals",
         "require": "visit.active"
     },
-
+    {
+        "id": "referenceapplication.realTime.admission",
+        "extensionPointId": "patientDashboard.visitActions",
+        "type": "link",
+        "label": "Admission Form",
+        "url": "htmlformentryui/htmlform/enterHtmlFormWithSimpleUi.page?patientId={{patient.uuid}}&visitId={{visit.uuid}}&definitionUiResource=file%3Aconfiguration%2Fhtmlforms%2Fauto-ipd-admission.xml",
```

Comment:
Same here with the encoded text.  Let's not pre-encode, but assume that the page rendering will encode correctly.